docs/zkapps/tutorials/03-deploying-to-a-network.mdx

```diff
@@ -20,167 +22,200 @@ zkApp programmability is not yet available on the Mina Mainnet. You can get star
 
-- If this is a testnet, request tMINA at:
-  https://faucet.minaprotocol.com/?address=<YOUR-ADDRESS>
-- To deploy, run: `zk deploy berkeley`
-```
+The `config.json` configuration file was automatically scaffolded when created your project with the `zk project` command. However, the generated file does not yet contain the deploy alias. 
 
-This command also generates keys, in `keys/berkeley.json`, that we will use in our application.
+You can have one or more deploy aliases for your project. Using naming patterns is helpful when you have more than one deploy alias. You can change the name at anytime in the `config.json` file.
 
-## Request funds from the faucet
+A deploy alias consists of:
 
-To deploy your zkApp, you will need some funds to pay for transaction fees.
+- A self-describing name. This tutorial uses `berkeley`. The deploy alias name can be anything and does not have to match the network name.
+- The Mina GraphQL API URL that defines the network that receives your deploy transaction and broadcast it to the appropriate Mina network (Testnet, Devnet, Mainnet, and so on)
+- The transaction fee (in MINA) to use when deploying
 
-To get funds on the Berkeley Testnet, use the URL that was shown from the CLI output. Visit `https://faucet.minaprotocol.com/?address=<YOUR-ADDRESS>` and click **Request**.
+The `zk config` command helps manage your deployment. The command prompts guide you to create or update a deploy alias in your project `config.json` file. 
 
-You will have to wait a few minutes for the next block to include your transaction, so you'll have tMINA before proceeding to the next step.
+1. To configure your deployment, run the `zk config` command and respond to the prompts:
 
-## Deploy your smart contract
+  ```sh
+  $ zk config
+  ```
 
-To deploy your smart contract to the network, run the following command:
+  For this tutorial on Berkeley Testnet, use:
+  
+    - name: `berkeley`
+    - Mina GraphQL API URL: `https://proxy.berkeley.minaexplorer.com/graphql`
+    - transaction fee: `0.1`
 
-```sh
-$ zk deploy berkeley
-```
+  Public and private keys to use in your application are automatically generated in `keys/berkeley.json`.
 
-When running the deploy command, the zkApp CLI will compute a verification key for your smart contract. Computing the verification key can take 1-2 minutes, so please be patient. The zkApp CLI will show you the details of the transaction such as the network name, the URL, and the smart contract that will be deployed.
+1. Request funds from the Testnet Faucet to fund your fee payer account.
 
-Finally, enter `yes` or `y` when prompted, to confirm and send the transaction.
+  Follow the prompts to request tMINA. For this tutorial, your MINA ADDRESS is populated on the Testnet Faucet. tMina arrives at your address when the next block is produced (~3 min).
 
-You will see the following output:
+1. To deploy your project:
 
-```sh
-$ zk deploy berkeley
+  ```sh
+  $ zk deploy berkeley
```

Comment:
If a user enters `zk deploy` without entering a deploy alias, an interactive prompt is displayed to select one of their deploy aliases. 

```
? Which deploy alias would you like to deploy to? …
❯ berkeley
  mainnet
```
Maybe we should highlight the interactive path `zk deploy` instead of the non-interactive flow `zk deploy <deploy alias>` to be consistent with the previous tutorials.